frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -46,9 +48,21 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
   onClose,
 }) => {
   const [modalOpen, setModalOpen] = React.useState<boolean>(false);
-  const quickStart = getQuickStartByName(activeQuickStartID);
+  const [quickStart, setQuickStart] = React.useState<QuickStart>(null);
   const [scrollDirection, handleScrollCallback] = useScrollDirection();
   const { t } = useTranslation();
+  const [qsData, qsLoaded, qsError] = useK8sWatchResource<QuickStart[]>({
+    kind: referenceForModel(QuickStartModel),
+    namespaced: false,
+    isList: true,
+  });
+
+  React.useEffect(() => {
+    if (qsData && qsLoaded && !qsError) {
+      const index = qsData.findIndex((qs) => qs.metadata.name === activeQuickStartID);
+      index > -1 && setQuickStart(qsData[index]);
+    }
+  }, [activeQuickStartID, qsData, qsLoaded, qsError]);
```

Comment:
Do not load the quick starts here directly. instead use `QuickStartsLoader` to load the quick starts.

Reply:
wrapped Drawer inside Loader